xdsl/parser/attribute_parser.py

```diff
@@ -184,24 +184,40 @@ def _parse_dialect_type_or_attribute_inner(
         The contents will be parsed by a user-defined parser, or by a generic parser
         if the dialect attribute/type is not registered.
         """
+        pretty = "." in attr_name
```

Comment:
is this format also called "pretty" in MLIR? Just making sure that there is no confusion in terminology between pretty/custom/etc.

Reply:
Yep!